documentation/source/user_section/tutorials/shape-analysis/compute-csa-and-other-shape-metrics/other-shape-metrics.rst

```diff
@@ -3,11 +3,31 @@
 Other shape metrics
 ###################
 
-The csv files generated by ``sct_process_segmentation`` also include metrics to analyse the shape of the spinal cord in the axial plane, such as ellipticity, antero-posterior and right-left dimensions. These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ for an example application in degenerative cervical myelopathy.
+The csv files generated by ``sct_process_segmentation`` include besides the cross-sectional area (CSA) also the following metrics to analyse the shape of the spinal cord in the axial plane:
+
+================  =======
+    CSV key       Meaning
+================  =======
+``diameter_AP``   Antero-posterior diameter (AP diameter)
+``diameter_RL``   Transverse diameter
+``eccentricity``  Eccentricity of the ellipse that has the same second-moments as the spinal cord. Eccentricity is the ratio of the focal distance over the major axis length. The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle.
+``solidity``      CSA(spinal_cord) / CSA_convex(spinal_cord). The more ellipse-shaped the cord is (i.e. the closer the perimeter of the cord is to being fully convex), the closer the solidity ratio will be to 1. This metric is interesting for detecting concave regions (e.g., in case of strong compression).
+``orientation``   Angle (in degrees) between the AP axis of the spinal cord and the AP axis of the image.
+================  =======
+
+These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ and `Horáková et al. Quant Imaging Med Surg 2022 <https://pubmed.ncbi.nlm.nih.gov/35371944/>`_ for an example applications.
+
+For an interactive database of normative values, see `Valošek, Bédard et al. NeuroLibre 2023 <https://neurolibre.org/papers/10.55458/neurolibre.00017>`_.
+
+----
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/shape-metric-computation/sct_process_segmentation-shape-metrics.png
    :align: center
 
+   Figure Caption: (A) Compression at the level of the intervertebral disc C5-C6 (red arrows) is manifested by (B) the decrease of AP Diameter, Compression Ratio, and Solidity and by the increase of Eccentricity (red arrows). The blue line represents normative values computed from 105 males from the spine-generic dataset. Adapted from `Valošek, Bédard et al. NeuroLibre 2023 <https://doi.org/10.55458/neurolibre.00017>`__.
```

Comment:
The caption is no longer relevant after the figure update (https://github.com/spinalcordtoolbox/doc-figures/pull/5). I believe that the figure is self-explanatory, and no caption is actually needed.

Reply:
Sounds good to me! I think it can be removed. :)

Reply:
Okay! Removed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4262/commits/ca8431dcf28b5f4f139c134c88311d5bd2499b9e.